examples/browser-rollup/package.json

```diff
@@ -10,6 +10,7 @@
     "uuid": "file:../../.local"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^11.0.2",
     "rollup": "^1.32.0",
     "rollup-plugin-node-resolve": "^5.2.0",
```

Comment:
`@rollup/plugin-node-resolve`

Reply:
@TrySound when I upgrade to `@rollup/plugin-node-resolve` the `{browser: true}` setting seems to be ignored. Here's the output of an `npm run build` inside `examples/browser-rollup`:

```
./example-all.js → dist/all.js...
(!) Plugin node-resolve: preferring built-in module 'crypto' over local alternative at 'crypto', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
crypto (imported by ../../dist/esm-node/sha1.js, ../../dist/esm-node/rng.js, ../../dist/esm-node/md5.js)
(!) Missing global variable name
Use output.globals to specify browser global variable names corresponding to external modules
crypto (guessing 'crypto')
created dist/all.js in 417ms
```

Any idea why?

Reply:
After an hour of debugging I figured out the problem is symlink to uuid. The new version does not handle it properly.

Reply:
@TrySound thanks for looking into this, you are of great help for this library!

Have you been able to figure out if that is an intended behavior of the new rollup plugin or a bug? If so, would you mind reporting it upstream with the information you discovered?

Reply:
I think it's a bug. I'll try to write test and fix the issue. Keep old package for now.

Reply:
By the way "preserveSymlinks: true" in rollup config solves the problem with new package.

Reply:
Fixed here https://github.com/rollup/plugins/pull/291